### ISSUE TITLE
After training your model not give the same test accuracy as reported in the Paper. 

### ISSUE THREAD

**Issue by Altaf-hucn** [FernandoJRS/violence-detection-deeplearning#3]
### Thanks for providing the training code. 
I have run your code and changed the number of epochs to 100 from 25. After successful training, your model does not give the same testing accuracy as reported in the paper.
